wai-extra/Network/Wai/Middleware/RequestLogger/JSON.hs

```diff
@@ -73,19 +76,26 @@ word32ToHostAddress = T.intercalate "." . map (T.pack . show) . fromIPv4 . fromH
 readAsDouble :: String -> Double
 readAsDouble = read
 
-requestToJSON :: NominalDiffTime -> Request -> [S8.ByteString] -> Value
-requestToJSON duration req reqBody =
-  object
+-- | Get the JSON output for a request
```

Comment:
This description isn't clear enough from my reading. I'm not sure what a "JSON output for a request" means. I think "JSON representation" is closer to the mark, but a little more clarity around this would definitely be helpful.

Reply:
I was struggling to describe this myself. I'll opt for "JSON representation" - but how would you describe *how* the representation is defined? I didn't want the doc comment to imply some kind of API for serialisation that users might then depend on.

Reply:
I've then spell that out. Clarify what _kind_ of information it includes (like method etc) but that the exact representation may change over time.

Reply:
Let me know what you think. I tried to be clear about the structure without being too verbose - there didn't seem to be a need to explain what most of the object fields did, for example.